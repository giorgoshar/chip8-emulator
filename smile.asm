.org   0x200
jmp    main
.ascii  "GREET.ASM, Copyright 2014 Boro Sitnikovski"
main:

SMILE:
    ; x position
    load    v1, 5

    ; y position
    load    v2, 5     

    ; load font    
    load    [I], l_SMILE

    ; draw
    draw    v1, v2, 5
loop:
    jmp     loop

.font l_SMILE 0x18 0x48 0x08 0x48 0x18, endfont
.font l_SAD   0x18 0x50 0x10 0x50 0x18, endfont