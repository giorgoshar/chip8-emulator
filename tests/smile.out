,GREET.ASM, Copyright 2014 Boro Sitnikovskiab¢6Ñ%4HHPP